addon/components/o-s-s/button-dropdown.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="oss-button-dropdown">
+<div class={{concat "oss-button-dropdown oss-button-dropdown--" this.buttonSize}}>
```

Comment:
readability nit:

replace it w/ `this.computedClasses`

and in the TS you can implement the getter to return the necessary classes ([example](https://github.com/upfluence/oss-components/blob/e4e6457c46e6a907273ee4902f5bd334a93b2b45/addon/components/o-s-s/pill.ts#L18))